_maps/icebox.json

```diff
@@ -1,8 +1,8 @@
 {
 	"version": 1,
 	"map_name": "Ice Box Station",
-	"map_path": "map_files/IceBoxStation",
-	"map_file": "IceBoxStation.dmm",
+	"map_path": "map_files/IceBoxStation_bubber",
```

Comment:
donut change map path, that is what folder, the server is looking for a directory called "IceBoxstation_bubber" which doesn't exist.